src/server/frontend_wayland/wl_data_device.cpp

```diff
@@ -49,24 +73,28 @@ class mf::WlDataDevice::Offer : public wayland::DataOffer
 
     void accept(uint32_t serial, std::optional<std::string> const& mime_type) override
     {
-        (void)serial, (void)mime_type;
+        (void)serial;
```

Comment:
I think I have a weak preference for `[[maybe_unused]]` annotations instead?

Reply:
Pre-existing. :grin:

There's also `uint32_t /*serial*/`